utils/constants.jsx

```diff
@@ -238,9 +238,12 @@ export const ActionTypes = keyMirror({
 
     EMOJI_POSTED: null,
 
-    RECEIVED_PLUGIN_COMPONENTS: null,
-    RECEIVED_PLUGIN_POST_TYPES: null,
-    RECEIVED_PLUGIN_MENU_ACTIONS: null,
+    RECEIVED_PLUGIN_COMPONENT: null,
+    REMOVED_PLUGIN_COMPONENT: null,
+    RECEIVED_PLUGIN_POST_COMPONENT: null,
+    REMOVED_PLUGIN_POST_COMPONENT: null,
+    RECEIVED_PLUGIN_MENU_ACTION: null,
```

Comment:
Are these two actually being used somewhere?

Reply:
Whoops, forgot to remove them